script.js

```diff
@@ -0,0 +1,133 @@
+const menuBtn = document.querySelector('.menu-btn');
+const navItem = document.querySelector('.item');
+const navItemBtn = document.querySelectorAll('.nav-item-btn');
+let menuOpen = false;
+
+function openMenu() {
+  menuBtn.classList.add('open');
+  navItem.classList.add('active-menu');
+  menuOpen = true;
+}
+
+function closeMenu() {
+  menuBtn.classList.remove('open');
+  navItem.classList.remove('active-menu');
+  menuOpen = false;
+}
+
+navItemBtn.forEach((itemBtn) => {
+  itemBtn.addEventListener('click', () => {
+    closeMenu();
+  });
+});
+
+menuBtn.addEventListener('click', () => {
+  if (!menuOpen) {
+    openMenu();
+  } else {
+    closeMenu();
+  }
+});
+
+function homePage() {
+  const performers = document.querySelector('.performers');
+  const moreBtn = document.getElementById('more');
+  const arrow = document.getElementById('arrow');
+  let more = false;
+  const performersInfo = [
+    {
+      image: 'img/cho.jpg',
+      altImage: 'Seong-Jin Cho Picture',
+      name: 'Seong-Jin Cho',
+      performersTitles: `South Korean pianist, 
+        winner of first prize at the 17th Warsaw International Chopin 
+        Piano Competition.`,
+      performersResume: `He displayed an impressive 
+        variety of tonal colors and remarkable technique, dispatched 
+        with jaw-dropping panache. He had something to say and 
+        communicated it well.`,
+    },
+    {
+      image: 'img/Liu_Kate_square.jpg',
+      altImage: 'Kate Liu picture',
+      name: 'Kate Liu',
+      performersTitles: `American classical pianist born 
+        in Singapore.`,
+      performersResume: `Pianist Kate Liu gained international 
+      acclaim after winning the Bronze Medal and Best Mazurka 
+      Prize at the 17th International Fryderyk Chopin Competition 
+      in Warsaw, Poland.`,
+    },
+    {
+      image: 'img/640px-Avdeevea.jpg',
+      altImage: 'Yulianna Avdeeva picture',
+      name: 'Yulianna Avdeeva',
+      performersTitles: `Yulianna Andreevna Avdeeva is 
+        a Russian concert pianist.`,
+      performersResume: `
+        She became famous by 
+        winning first prize at the 16th edition of the Frédéric Chopin 
+        International Piano Competition, the first woman to win it 
+        since Martha Argerich, in 1965.`,
+    },
+    {
+      image: 'img/rafal-blechacz.jpg',
```

Comment:
**OPTIONAL** - According to the project requirements, this section should have six articles as it is in the design template. Please fix this 🙏. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your project:
<img width="940" alt="wvqebe" src="https://user-images.githubusercontent.com/73607512/203840865-61973ab5-9c2b-43a5-9b7f-fcfce91451a6.PNG">